### ISSUE TITLE
Should envvar prefix use the class name?

### ISSUE THREAD

**Issue by python-clout** [python-clout/clout#9]
I'm not sure if the environment variable prefix should use the class name or not.

```py

import clout


@attr.dataclass
class User:
    name: str


@attr.dataclass
class DB:
    host: str
    port: int
    user: User


@attr.dataclass
class Config:
    db: DB
    debug: bool
    user: User
    priority: float = attr.ib(
        default=0,
        metadata={
            "clout": {
                "cli": dict(param_decls=["--priority"], help="App priority value")
            }
        },
    )
    logging: bool = True
    dry_run: bool = False


APP_NAME = "myapp"


# Read from environment_variables prefixed `MYAPP_`,
# such as MYAPP_CONFIG_PRIORITY=10.
# XXX Would it be better as `MYAPP_PRIORITY` without inferring basd on the class name?
ENVVAR_DATA = clout.load_env(Config, prefix=APP_NAME)


```

**Comment by python-clout** [comment]
I think it should not. That's too much magic and it's easy to specify the class name in `prefix=` if you want it.

**Comment by python-clout** [comment]
Resolved in e74da6c
